1.0.0/reference.json

```diff
@@ -247,6 +247,50 @@
         "expression": true,
         "type": "float"
       }
+    },
+    "dot": {
+      "opacity": {
+        "css": "dot-opacity",
+        "doc": "The overall opacity of the dot.",
+        "default-value": 1,
+        "default-meaning": "The dot opacityis 1.",
+        "type": "float"
+      },
+      "fill": {
+        "css": "dot-fill",
+        "default-value": "#FFF",
+        "doc": "The color of the area of the dot.",
+        "type": "color",
+        "expression": true
+      },
+      "width": {
+        "css": "dot-width",
+        "default-value": 10.0,
+        "doc": "The width of the dot, if using one of the default types.",
+        "type": "float",
+        "expression": true,
+        "default-meaning": "The dot width is 10 pixels."
+      },
+      "height": {
+        "css": "dot-height",
+        "default-value": 10.0,
+        "doc": "The height of the dot, if using one of the default types.",
+        "type": "float",
+        "expression": true,
+        "default-meaning": "The dot height is 10 pixels."
+      },
+      "comp-op": {
```

Comment:
not sure if we support (or will be able to) comp-op in tangram, is that possible?